msystems/services.py

```diff
@@ -56,5 +78,45 @@ def _update_user(self, user: User, user_data: dict) -> None:
             user.i_user.save()
 
     def _update_user_legal_entities(self, user: User, user_data: dict) -> None:
-        # TODO Handling of legal entities
-        pass
+        legal_entities = self._parse_legal_entities(user_data.get('OrganizationAdministrator'))
+        policyholders = [self._get_or_create_policy_holder(user, line[1], line[0]) for line in legal_entities]
+
+        self._delete_old_user_policyholders(user, policyholders)
+        self._add_new_user_policyholders(user, policyholders)
+
+    def _parse_legal_entities(self, legal_entities) -> map:
+        return map(lambda s: s.rsplit(' ', 1), legal_entities)
```

Comment:
It'd be good to add a comment explaining format of administrative organization string

Reply:
I added an explanation